Makefile

```diff
@@ -134,6 +138,17 @@ test-integration: manifests generate fmt vet envtest ginkgo ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" \
 	$(GINKGO) -v $(INTEGRATION_TARGET)
 
+.PHONY: test-e2e
+test-e2e: manifests generate fmt vet envtest ginkgo kind-delete kind-create e2e-docker image-load
```

Comment:
we can run `deploy` cmd directly, no need to apply `test/e2e/config`, then we can remove these configuration files.

Reply:
Can't we apply `imagePullPolicy=IfNotPresent` in `config/default/manager_webhook_patch.yaml`?

Reply:
https://github.com/kubernetes-sigs/kueue/pull/421#discussion_r999846658

@tenzen-y and @alculquicondor requested that I look into kustomization to configure imagePullPolicy.  I am not very familar with kustomize so I actually would prefer to just modify the default to be IfNotPresent but I am unsure if that would be allowed.  

This is why I went with a separate config for the e2e tests.  I'm open to removing the test/e2e/config and just setting our default configs to IfNotPresent if @alculquicondor or @tenzen-y are okay with that.

Reply:
I believe that `imagePullPolicy=Always` is more helpful for the users. So I suggested overwriting imagePullPolicy using the kustomize patches.

Reply:
You can try like this, it worked out for me. https://github.com/kubernetes-sigs/kueue/commit/5f2b21318a1357b11bc65c31c68a9b6078c98bc2

Reply:
@kerthcet Thank you for that! Pushed up the fix.